packages/react-components/react-progress/Spec.md

```diff
@@ -2,62 +2,105 @@
 
 ## Background
 
-_Description and use cases of this component_
+A `Progress` is a component that is used to display the current progress of an operation flow.
 
 ## Prior Art
 
-_Include background research done for this component_
+### Open UI
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+| Library    | Component Name | Spec Link                                                          | Notes                                                                   |
+| ---------- | -------------- | ------------------------------------------------------------------ | ----------------------------------------------------------------------- |
+| Ant Design | Progress       | [Progress](https://ant.design/components/progress/)                | Specifies the type, a combined progress component                       |
+| Bootstrap  | Progress       | [Progress](https://getbootstrap.com/docs/4.3/components/progress/) | Allows for multiple different animation styles on the same Progress bar |
+
+### Comparison of v8 and v0
+
+The existing components are:
+
+- v8
+  - `ProgressIndicator`
+- v0
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+Basic examples:
+
+```jsx=
+import { Progress } from "@fluentui/react-progress";
+
+function App() {
+    return (
+        <Progress barHeight="medium" label="Loading" />
+    )
+}
+
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+- Indeterminate Progress
+  - The default Progress that animates indefinitely
+- Determinate Progress
+  - The determinate form of the Progress component that incrementally loads from 0 to 100
+
+### Shape
+
+The Progress is represented as a rectangular area with an inner animated bar that either travels across the area indefinitely or animates up till a specified point
 
 ## API
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+### Slots
+
+- `root` - The root element of the Progress. The html element is a `div`
+- `indicator` - The div element that gets animated into a Progress. The html element is `span`
+- `label` - The text shown above the Progress. This uses the `Label` control
+- `description` - The text shown below the Progress. This uses the `Label` control
+
+### Props
+
+See API at [Progress.types.tsx](./src/components/Progress/Progress.types.ts).
```

Comment:
FYI: it's currently empty:

![image](https://user-images.githubusercontent.com/14183168/186392360-2e50fdc0-8cce-42c7-b77b-3933f56e169e.png)

It would be great to include currently implemented props from #24023 to the spec.